_includes/charts/calico/templates/calico-typha.yaml

```diff
@@ -74,6 +74,19 @@ spec:
           name: calico-typha
           protocol: TCP
         env:
+{{- if .Values.bpf }}
```

Comment:
Why have this if Typha is not supported with BPF?

Reply:
Just planning ahead, so when we do add typha it should "just work".  Figured it'd be easy to miss this one when we came to do that.